shotover-proxy/src/message/mod.rs

```diff
@@ -426,45 +423,113 @@ impl Value {
     }
 
     pub fn build_value_from_cstar_col_type(spec: &ColSpec, data: &CBytes) -> Value {
-        if let Some(actual_bytes) = data.as_slice() {
```

Comment:
No longer need this check because cdrs-tokio will make the type CassandraType::Null